### ISSUE TITLE
Should ignore HTML slashes

### ISSUE THREAD

**Issue by BrodaNoel** [yannickcr/eslint-plugin-react#1270]
Hi guys!

I'm getting:
```bash
./src/pages/Admin/Campaigns/NewReaction.jsx
  Line 98:  Comments inside children section of tag should be placed inside braces  react/jsx-no-comment-textnodes

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

This is my line 98:
```jsx
            <span className="pl-c"><span className="pl-c">&#47;&#47;</span> ...</span><br />
```

Those `&#47;` are literally wrote in that way.

See you!

**Comment by ljharb** [comment]
The proper solution is `{'//'}`, as opposed to encoding entities directly.

**Comment by giraffesyo** [comment]
That works perfectly, but I still agree that `&#47;` should work as well.

**Comment by ljharb** [comment]
I think that the common convention to indicate intentionality in jsx string literals is with curlies; we could also support using html encodings, but i think that’s also likely to be an error, like //.

**Comment by ljharb** [comment]
Looks like it works in the latest version.

**Comment by BrodaNoel** [comment]
Woho! Thanks!
I already changed company 2 times, but still a win! Thank you a lot!
